[quantization] Remove duplicated piece of code in test (just a nit).

radkris-git opened the pull request:
Summary: Remove duplicated piece of code in test (just a nit).

Test Plan: buck test test:quantization

Differential Revision: D21967877

jerryzh168 reviewed (approved):
thanks

jerryzh168 commented on test/quantization/test_quantized_op.py:
```diff
@@ -2048,8 +2048,6 @@ def test_qlinear(self, batch_size, input_channels, output_channels,
                         (X_value_max - X_value_min)
                         + X_value_min
                         ).astype(np.uint8)
-        X_q0 = np.round(np.random.rand(batch_size, input_channels) *
```
I feel formatting for this one is actually better :)

Commit:
[quantization] Remove duplicated piece of code in test (just a nit). (#39770)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/39770

Remove duplicated piece of code in test (just a nit).

Test Plan: buck test test:quantization

Differential Revision: D21967877

fbshipit-source-id: e9150c8749a5048d02706fd686e21bfac06c767b
